5/UnitTests/src/UnitTests/Tests.fs

```diff
@@ -0,0 +1,46 @@
+module UnitTests.Tests
+
+open Xunit
+open Library
+open Interfaces
+open System.Threading.Tasks
+open Foq
+open Client
+
+[<Fact>]
+let ``Library Valid String Input Test`` () =
+    let result = Say.hello("Mereta")
+    Assert.Equal("Hello, \"Mereta\"! Your name is 6 characters long.", result)
+
+[<Fact>]
+let ``Library Empty String Input Throws Exception`` () =
+    let methodCall = (fun () -> Say.hello "" |> ignore)
```

Comment:
Technically it's not a method call, but a function (uncalled yet), so maybe it would make more sense to either extract it as a reusable function or as an inline lambda used in lines e.g. 18, 24, etc.